pkg/sync/v6/data/ConfigMap/openshift-azure-monitoring/metrics-bridge.yaml

```diff
@@ -186,6 +186,11 @@ data:
       name: grpc_server_handled_rate
     - query: 'etcd_disk_backend_commit_duration_seconds_count{namespace="openshift-azure-monitoring"} != 0'
     - query: 'etcd_disk_backend_commit_duration_seconds_sum{namespace="openshift-azure-monitoring"} != 0'
+    - query: 'sync_info'
+    - query: 'rate(sync_errors_total[5m])'
+      name: sync_errors_rate
+    - query: 'rate(sync_duration_seconds_sum[5m]) / rate(sync_duration_seconds_count[5m])'
```

Comment:
Isn't `sync_duration_seconds` a summary metric? Why not forward the whole timeseries (which will include p50, p90, p99 by default I think) rather than manually calculating the average?

Reply:
We tried sending the summary metrics and there's no good way to present the data in Geneva. 

For alerting we wanted recent data so last run or an average of the last 2 runs in the worst case. Sending the metric with quantile=0.5 would show us the 50th percentile since the start of the sync pod which would grow much slower if sync runs suddenly took minutes instead of the typical 6s.

Reply:
Hm, the median will grow much slower but p90/p99 won't. Mostly I'm just wary of further aggregating this summary data into an average because averages tend to be very lossy.

Reply:
@ehashman summary quantiles should generally be avoided as you cannot do math such as averages on top of quantiles, which prevents one from aggregating client-side quantiles from across instances of a given service. 

That and the fact that it is better to send 1 time series instead of 5 series to geneva are the reasons why i chose not to forward 5 of them and do the precalculation.

Reply:
> summary quantiles should generally be avoided as you cannot do math such as averages on top of quantiles, which prevents one from aggregating client-side quantiles from across instances of a given service.

This is also true of sending a precalculated average as we're doing here though... if this is a concern we should be storing this as a histogram (although that doesn't solve the concern of sending more time series to Geneva). I just generally don't think averages are useful metrics, they hide too much info.